### PR TITLE
Buzzinga test

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: tests/buzzinga_test/main.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+    student "student"
+
+    "github.com/01-edu/go-tests/lib/challenge"
+    "github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
```
Comment (Hamzaelkhatri) [inline tests/buzzinga_test/main.go:10]: add some test for negative numbers

**Review by Hamzaelkhatri — APPROVED** [review]
